tests/test_helpers.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import numpy as np
+
+from . import PandasTestCase
+import doctest
+import unittest
+import warnings
+
+from texthero import _helper
+
+"""
+Doctests.
+"""
+
+
+def load_tests(loader, tests, ignore):
+    tests.addTests(doctest.DocTestSuite(_helper))
+    return tests
+
+
+"""
+Test Decorators.
+"""
+
+
+class TestNLP(PandasTestCase):
```

Comment:
Rename `TestNLP `  to something more appropriate?

Reply:
Wow, not sure how we missed that! :roll_eyes:  It's renamed!